WAW.API/Chat/Domain/Models/ChatRoom.cs

```diff
@@ -0,0 +1,10 @@
+using WAW.API.Shared.Domain.Model;
+
+namespace WAW.API.Chat.Domain.Models;
+
+public class ChatRoom : BaseModel {
+  public DateTime Date { get; set; }
```

Comment:
I think a `DateTime` with the `CreationDate` and another one with the `LastUpdateDate` would be better.